CONTRIBUTING.md

```diff
@@ -49,8 +48,12 @@ Installation instructions can be found here: https://yarnpkg.com/en/docs/install
 
 ### Setup
 
+Fork the `babel` repository to your Github Account.
```

Comment:
```suggestion
Fork the `babel` repository to your GitHub Account.
```

Reply:
Sure, making the changes as soon as I get how to undo the formatting. Thanks.